packages/user-mgnt/src/config/routes.ts

```diff
@@ -203,16 +203,7 @@ export const getRoutes = () => {
         tags: ['api'],
         description: 'Changes password for logged-in user',
         auth: {
-          scope: [
-            SCOPES.RECORD_DECLARE_BIRTH,
-            SCOPES.RECORD_DECLARE_DEATH,
-            SCOPES.RECORD_DECLARE_MARRIAGE,
-            SCOPES.REGISTER,
-            SCOPES.CERTIFY,
-            SCOPES.PERFORMANCE_READ,
-            SCOPES.CONFIG_UPDATE_ALL,
-            SCOPES.RECORD_SUBMIT_FOR_APPROVAL
-          ]
+          scope: [SCOPES.USER_UPDATE]
```

Comment:
This route is by a user to change their own password, so we shouldn't impose any scope restrictions here